src/kit/DataGrid/menu.tsx

```diff
@@ -33,23 +33,23 @@ export interface HeaderMenuProps {
 export const HeaderMenu: React.FC<HeaderMenuProps> = ({ bounds, open, handleClose, items }) => {
   const divRef = useRef<HTMLDivElement | null>(null);
   useOutsideClickHandler(divRef, handleClose);
-  return (
-    <Dropdown autoWidthOverlay menu={items} open={open} placement="bottomLeft">
-      <div
-        ref={divRef}
-        style={
-          open
-            ? {
-                height: bounds.height,
-                left: bounds.x,
-                position: 'fixed',
-                top: bounds.y,
-                width: bounds.width,
-              }
-            : {}
-        }
-        onClick={handleClose}
-      />
-    </Dropdown>
-  );
+  if (open) {
+    return (
+      <Dropdown autoWidthOverlay menu={items} open placement="bottomLeft">
```

Comment:
if other ppl are fine with this approach, i wouldnt block approval, but not sure if its the ideal solution even though that fixes the issue.  looks like css doing something wrong. cant we fix the issue with css (style)?

Reply:
updated with a different approach that still uses `Dropdown`'s `open` prop and changes the custom `style` prop value instead.